aiosmtpd/docs/conf.py

```diff
@@ -15,14 +15,13 @@
 import sys
 import re
 import datetime
+import sphinx_rtd_theme  # noqa: F401
```

Comment:
`# noqa` needed because this import is not used within `conf.py`, but RTD documentation explicitly specifies that we need to put this import in `conf.py`.